### ISSUE TITLE
pbiviz --install-cert Doesn't Run on 3.1.11

### ISSUE THREAD

**Issue by dm-p** [microsoft/PowerBI-visuals-tools#306]
Hi folks, and thanks so much in turning around #304 so quickly :)

I can confirm that dependencies now work as expected, but if you run `pbiviz --install-cert`, it does not perform the action of installing the generated certificate any more, e.g.:

````
PS C:\Repos\test> pbiviz --install-cert
Usage: pbiviz [options] [command]

Options:
  -V, --version     output the version number
  --install-cert    Creates and installs localhost certificate
  -h, --help        display help for command

Commands:
  new [name]        Create a new visual
  info              Display info about the current visual
  start             Start the current visual
  package           Package the current visual into a pbiviz file
  update [version]  Updates the api definitions and schemas in the current visual. Changes the version if specified
  help [command]    display help for command
````

## Workaround:
* If a valid certificate is generated through the first run of `pbiviz start`, it needs to be manually installed via explorer by navigating to the `node_modules/certs` folder and double-clicking it.
* If the developer does not note the passcode on the initial `pbiviz start` they will not have it available for the above step; if this is the case, they will need to:
    * Manually delete the certificate from the folder.
    * Re-run `pbiviz start`, noting the passphrase in the output.
    * Repeat the manual installation step from above.

I've tried this on 2 different machines (both installing over 3.1.10) with the same result. I've also tried uninstalling the module and starting again, but the result is also the same, unfortunately.

**Comment by zBritva** [comment]
@dm-p, I think updating the package "commander" to 5.0.0 is reason. I reverted it to the version 4.1.1, now it works expected. @EugeneElkin could you check and confirm?

**Comment by EugeneElkin** [comment]
I will

**Comment by uve** [comment]
@dm-p @zBritva fixed in powerbi-visuals-tools@3.1.12
